Mathlib/Analysis/Calculus/BumpFunction/FiniteDimension.lean

```diff
@@ -142,10 +142,8 @@ theorem IsOpen.exists_smooth_support_eq {s : Set E} (hs : IsOpen s) :
       refine le_trans ?_ (le_max_left _ _)
       apply Finset.le_max'
       apply Finset.mem_image_of_mem
-      -- Porting note: was
-      -- simp only [Finset.mem_range]
-      -- linarith
-      simpa only [Finset.mem_range, Nat.lt_add_one_iff]
+      simp only [Finset.mem_range]
+      linarith
```

Comment:
`linarith` seems pretty heavy for what it needs to do here. No objection if you prefer it, though

Reply:
Switched to `omega`. I never do arithmetic by hand if I can help it, even if it's a single step. :-)